src/System.Net.Utilities/ref/System.Net.Utilities.cs

```diff
@@ -37,14 +37,6 @@ public enum IPStatus
     public partial class Ping
     {
         public Ping() { }
-        public System.Net.NetworkInformation.PingReply Send(System.Net.IPAddress address) { return default(System.Net.NetworkInformation.PingReply); }
```

Comment:
@SidharthNabar @WesHaggard @EricStJ @terrajobst: FYI: Removing _sync_ APIs from S.N.Utilities. Async versions are available and should be used instead.
This first version of the contract is currently marked as experimental and has never been shipped before.